src/Courses/CourseDetailsTabs/StatisticsTab.tsx

```diff
@@ -497,63 +517,68 @@ export const StatisticsTab: React.FC<StatisticsTabProps> = ({ course, userId })
                         </>)}
                     />
                 </>}
-                <MaterialTable
-                    icons={icons}
-                    title={(
-                        <div className="d-flex">
-                            <h6
-                                style={{
-                                    whiteSpace: 'nowrap',
-                                    overflow: 'hidden',
-                                    textOverflow: 'ellipsis'
-                                }}
-                                className="MuiTypography-root MuiTypography-h6"
-                            >
-                                {getTitle()}
-                            </h6>
-                            {userType === UserRole.PROFESSOR && !_.isNil(userId) && !_.isNil(grade) && (view === StatisticsViewFilter.PROBLEMS_FILTERED) && 
-                            <>
-                                <Button
-                                    className="ml-3 mr-1"
-                                    onClick={() => setGradesState({
-                                        ...gradesState,
-                                        view: GradesStateView.OVERRIDE
-                                    })}
-                                >
-                                    <>
-                                        <BsPencilSquare/> Override
-                                    </>
-                                </Button>
-
-                                <Button
-                                    variant={grade.locked ? 'warning' : 'danger'}
-                                    className="ml-1 mr-1"
-                                    onClick={() => setGradesState({
-                                        ...gradesState,
-                                        view: GradesStateView.LOCK
-                                    })}
+                { loading ? 
+                    <Spinner animation='border' role='status'><span className='sr-only'>Loading...</span></Spinner> :
```

Comment:
Didn't @PanopticaRising add a global loading spinner?

Reply:
swap to material-ui CircularProgress

Reply:
@red-tt-bear I never added the functionality for it to actually function globally.